tests/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ if(TARGET ${QT_VERSIONED_PREFIX}::QuickTest)
 endif()
 add_subdirectory("test_protobuf_multifile")
 add_subdirectory("test_extra_namespace")
+add_subdirectory("test_qmakebuild")
```

Comment:
I'm not quite sure if the test will work without QtProtobuf installer in the system.
I'm referring the https://github.com/semlanik/qtprotobuf/pull/225/files#diff-2167d9f28c33778cbc1f40d89aef2642ac5a795779df44ed6d07852fc5049758R2 . AFAIK this doesn't work if .pri files are not inside mkspec directory.

So my suggestion is to put the test into
```cmake
if(QT_PROTOBUF_STANDALONE_TESTS)
    add_subdirectory("test_qmakebuild")
endif()
```

Reply:
I'll propose another option in my next PR.